### ISSUE TITLE
False positive?

### ISSUE THREAD

**Issue by exploresecurity** [erwanlr/Fingerprinter#42]
Running a scan with `--app-name umbraco --unique-fingerprint` gave the following results:
```
Unique Match! v4.11.10 - https://{REDACTED}/umbraco_client/tablesorting/jquery.tablesorter.min.js -> 338986ae320233de0c83c5673828f8ca         
Unique Match! v4.11.10 - https://{REDACTED}/umbraco_client/Tree/treeIcons.css -> ca355d5390649ca2a30b83d8d654f544                             
Unique Match! v4.11.10 - https://{REDACTED}/umbraco_client/ui/knockout.mapping.js -> 612bb55c00823971bbaf8890b86cce18                         
Unique Match! v4.11.10 - https://{REDACTED}/umbraco_client/ui/jquery.tooltip.min.js -> f0dc3f8fce415d66fa44e81dbb9f9c22                       
Unique Match! v4.11.10 - https://{REDACTED}/umbraco_client/ui/base2.js -> ee40e4d5121a3439efded019ada661c1                                    
Unique Match! v4.11.10 - https://{REDACTED}/umbraco_client/ui/dd.css -> 8b4d925d17dae6813ba80deb68415fe1                                      
Unique Match! v4.11.10 - https://{REDACTED}/umbraco_client/ui/knockout.js -> 235475c7c3dc43c7cb7f6125be536c32                                 
Unique Match! v4.11.10 - https://{REDACTED}/umbraco_client/Application/JQuery/jquery.hotkeys.js -> f33d295fe69e3c013132eba0e1181d70           
Unique Match! v4.11.10 - https://{REDACTED}/umbraco_client/Application/JQuery/jquery.metadata.min.js -> 2131893ba70ac794a1d7a9ac228f0587      
Fingerprinting - Time: 00:04:34 <===================================================================================> (2262 / 2262) 100.00% Time: 00:04:34

Very likely to be v4.11.10
```
Although the conclusion is "very likely" - i.e. not 100% - all the "unique matches" point to 4.11.10. However, I've been told the Umbraco version is in fact 7.12.4 - and they sent me their packages.config file to prove it.

**Comment by erwanlr** [comment]
Try with non unique fingerprinting (`-f / --fingerprint`) as the 7.12.4 only has two unique fingerprints (which might be blocked in your case):

```
$ ./fingerprinter.rb -a umbraco --luf 7.12.4
Results for 7.12.4:
5ca44816aeb6a606a85e42fb4178415d bin/Umbraco.Core.xml
6f247960f4817b6d5ba324c62945d0ad bin/umbraco.xml
```

The unique detected are likely to be old files which have not been removed after upgrading unfortunately :/

**Comment by exploresecurity** [comment]
Thanks for quick reply. I had previously run a full fingerprint scan and it came up with loads of possibilities, although curiously none of the 129 "arrays" included both 7.12.4 and 4.11.10. Because of the timeout changes I made (issue 41) I can see that the requests for the above two files did not time out. But when I GET /bin/Umbraco.Core.xml and /bin/umbraco.xml manually I get 404s anyway (there's no preceding base path is there?) so no wonder 7.12.4 never came up. No worries - but thanks for tip about how upgrades are likely to confuse results.
